File: MODUL-2/python/PRAK205-2310817210007-RaymondHariyono.py
import math
print("Input")
a,b = map(int, input().split())

alas = int(math.sqrt(b**2 - a**2))
tinggi = a
keliling = a + b + alas
luas = 0.5 * alas * tinggi

print()
print("Output")
print(f"Alas = {alas}")
print(f"Tinggi = {tinggi}")
print(f"Keliling = {keliling}")
print(f"Luas = {luas:.0f} cm^2")

File: MODUL-2/bahasa.c/PRAK205-2310817210007-RaymondHariyono.c
#include <stdio.h>
#include <math.h>
int main(){
int a, b;

printf("Input \n");
scanf("%d",& a);
scanf("%d",& b);

int alas = sqrt(pow(b,2)-(pow(a,2)));
int tinggi = a;
int keliling = a + b + alas;
int luas = 0.5 * alas * tinggi;

printf("\n");
printf("Output \n");

printf("Alas= %d cm \n", alas);
printf("Tinggi= %d cm \n", tinggi);
printf("keliling= %d cm \n", keliling);
printf("Luas= %d cm^2 \n", luas);

return 0;
}